reframe/utility/sanity.py

```diff
@@ -776,3 +777,35 @@ def iglob(pathname, recursive=False):
     '''Replacement for the :func:`glob.iglob() <python:glob.iglob>`
     function.'''
     return pyglob.iglob(pathname, recursive=recursive)
+
+
+@deferrable
+def path_exists(path):
+    '''Replacement for the :func:`os.path.exists() <python:os.path.exists>`
+       function.
+    '''
+    return os.path.exists(path)
+
+
+@deferrable
+def path_isdir(path):
+    '''Replacement for the :func:`os.path.isdir() <python:os.path.isdir>`
+       function.
```

Comment:
```suggestion
    '''Replacement for the :func:`os.path.exists` function.
```

Reply:
And similarly for the rest. Can you also do the same for the `glob` function?

Reply:
Do you mean the `glob.glob` function?

Reply:
Because those are already there.

Reply:
Yes, but I mean to fix the link in the docstring. Because you copied-pasted the format from there and it was wrong. It didn't create a hyperlink. You have to write as I propose here.